hack/images/dcos/README.md

```diff
@@ -23,5 +22,42 @@
 - [x] s6 and k8s require files to exist in certain places on the host. this is terrible practice
   - mentioned this article to elizabeth, she has similar concerns with the HDFS framework
     - http://www.ibm.com/developerworks/library/l-mount-namespaces/
-- [ ] additional ports (e.g. 6443/apiserver) should be configurable
+- [x] additional ports (e.g. 6443/apiserver) should be configurable
 - [ ] support secure (TLS) apiserver
+
+## Walkthrough w/ the guestbook examples
+
+Some prerequisites:
+
+- set up a DCOS cluster
+- install the `kubernetes` app using the DCOS CLI tooling
+  - `dcos package install kubernetes`
+- build yourself a kubectl, either using the Kubernetes or Kubernetes-Mesos build process
```

Comment:
In the future for DCOS, we could build a simple service that exposes a correctly-configured `kubectl` using something like [hterm](https://github.com/chromium/hterm).  Then we could distribute a more portable thin client CLI.  cc @jsancio for current plans or standardizing a way to do this.